### DISCUSSION TITLE
How to run new config from app.py

### DISCUSSION THREAD

**Discussion by hargums** [conor19w/Binance-Futures-Trading-Bot#29] [General]
Hallo sir, i tried your software and trying to adapt it. Now i had some loses with current setting, how i can re run new config from app.py without create new trade from active trade before (i give long stop loss and now i had some floating losses)
Example :
I run bot with 7 trades, all coin trading with goldencross strategy with 10 lev

Then i want to change my running bot config with 8 trades, specify coin trading with triple ema strategy with 20 lev

Is it possible?
I scared my 7 previous active trade with floating loss from config before added with 8 additional trading from new config

**Comment by conor19w** [comment]
Hello Sir, Not sure what you mean
